modules/hcp-requirements-platform-version.adoc

```diff
@@ -13,26 +13,26 @@ The following table indicates which {product-title} versions are supported for e
 |Platform |Hosting {product-title} version |Hosted {product-title} version
 
 |{aws-full}
-|4.11 - 4.16
-|4.14 - 4.16 (only)
+|4.11 - 4.17
+|4.14 - 4.17 (only)
 
 |{ibm-power-title}
-|4.16
-|4.16 (only)
+|4.17
+|4.17 (only)
 
 |{ibm-z-title}
-|4.16
-|4.16 (only)
+|4.17
+|4.17 (only)
 
 |{VirtProductName}
-|4.14 - 4.16
-|4.14 - 4.16 (only)
+|4.14 - 4.17
+|4.14 - 4.17 (only)
 
 |Bare metal
-|4.14 - 4.16
-|4.14 - 4.16 (only)
+|4.14 - 4.17
+|4.14 - 4.17 (only)
 
-|Non-bare metal agent machines
-|4.16
-|4.16 (only)
-|===
+|Non-bare-metal agent machines
```

Comment:
no need to mention technology preview platform here?

Reply:
Good point. I'll add "(Technology Preview)".